streams/src/main/java/org/apache/kafka/streams/state/RocksDBConfigSetter.java

```diff
@@ -37,4 +37,20 @@
      * @param configs       the configuration supplied to {@link org.apache.kafka.streams.StreamsConfig}
      */
     void setConfig(final String storeName, final Options options, final Map<String, Object> configs);
+
+    /**
+     * This method should close any user-constructed objects that inherit from {@code org.rocksdb.RocksObject}
+     *
+     * Any such object created with @{code new} in {@link #setConfig} should have {@code close} called on it here to
```

Comment:
And then here 
```
Any object created with @{code new} in {@link #setConfig} and that inherits from {@code org.rocksdb.RocksObject} should have {@code close()} called on it here to
```

Reply:
Sounds good to me. Thanks for reviewing @cadonna !

Reply:
Use `@link` and also link to `should have {@link RocksObject#close() close()} called`

Reply:
Ack